setup.cfg

```diff
@@ -0,0 +1,135 @@
+# This file is used to configure your project.
+# Read more about the various options under:
+# https://setuptools.pypa.io/en/latest/userguide/declarative_config.html
+# https://setuptools.pypa.io/en/latest/references/keywords.html
+
+[metadata]
+name = teuthology-api
+description = A REST API to execute teuthology commands
+author = Vallari Agrawal
+author_email = val.agl002@gmail.com
+license = MIT
+license_files = LICENSE.txt
+long_description = file: README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+url = https://github.com/pyscaffold/pyscaffold/
+# Add here related links, for example:
+project_urls =
+    Documentation = https://teuthology-api.readthedocs.io/
```

Comment:
we wouldn't be deploying `docs/` dir content on RtD, right? because fastapi provides documentation with swagger UI at /docs endpoint

Reply:
I don't think it's necessarily a problem to publish docs on RtD. People have got to find out somehow that they can use the `/docs/` endpoint, and ideally we'd have API docs published that don't require running code to view. If we figured that out, we could link to them from RtD. What do you think?

Reply:
That's a good idea. Along with that, we can also use RtD docs to describe more things like user flow or anything else that needs documenting. 
For providing non-interactive docs, fastapi provides an alternative which is enabled by default at `/redoc` endpoint: https://fastapi.tiangolo.com/tutorial/first-steps/#alternative-api-docs
In production mode, we can disable interactive swagger UI `/docs` and change url for redoc documentation using https://fastapi.tiangolo.com/tutorial/metadata/#docs-urls